pkg/artifactory/provider.go

```diff
@@ -32,6 +32,7 @@ func Provider() *schema.Provider {
 		"artifactory_remote_cargo_repository":     resourceArtifactoryRemoteCargoRepository(),
 		"artifactory_remote_bower_repository":     resourceArtifactoryRemoteBowerRepository(),
 		"artifactory_remote_go_repository":        resourceArtifactoryRemoteGoRepository(),
+		"artifactory_remote_vcs_repository":       resourceArtifactoryRemoteVcsRepository(),
```

Comment:
Not a blocker. My OCD wants to alpha sort this list of resource names!

Reply:
No problem, sorted